lib/Dialect/LLHD/IR/LLHDOps.cpp

```diff
@@ -122,7 +123,7 @@ struct constant_int_all_ones_matcher {
 
 } // anonymous namespace
 
-unsigned mlir::llhd::getLLHDTypeWidth(Type type) {
+unsigned llhd::getLLHDTypeWidth(Type type) {
```

Comment:
Should this be circt::llhd, for clarity?

Reply:
Ah yep, I will keep the fully qualified `circt::llhd` namespaces where they used to be `mlir::llhd`.